e2e/cleanup_test.go

```diff
@@ -276,4 +274,14 @@ spec:
 		stdout, stderr, err := kubectl("delete", "ns", cleanupTest)
 		Expect(err).ShouldNot(HaveOccurred(), "stdout=%s, stderr=%s", stdout, stderr)
 	})
+
+	It("should delete unnecessary LVs", func() {
```

Comment:
I think this is not a test but a cleanup logic.
If so, use [AfterSuite](https://onsi.github.io/ginkgo/#global-setup-and-teardown-beforesuite-and-aftersuite) or something instead.